onboarding/password-management.md

```diff
@@ -0,0 +1,59 @@
+# Password management
+
+By working in the laboratory you will be close to very sensitive information such as secure access keys and passwords, for personal or group use, allowing you to gain access to servers or critical physical spaces.
+
+In this context, it is everyone's duty to ensure the security of these sensitive information throughout its use and you should take this responsibility in account in all your actions and decisions.
+
+For example:
+
+ - Taking a picture inside the lab can lead to sensible information leaking on social's network due to some computer screen, note or post-it in the background of the picture
+ - Committing files on git without reread can lead to sensible information leaking on the cloud due to config file containing password or keys 
+
+Although we cannot have total security, simple practices and awareness of laboratory members can already avoid a lot of problems.
+
+## Storage of the sensitive information
+
+Polytechnique Montreal does not provide a secure password system within the university, this responsibility is therefore distributed to each member of the laboratory.
```

Comment:
This can be made more persausive

```suggestion
Polytechnique Montreal does not provide a secure password system within the university, but NeuroPoly expects members of the laboratory will use password managers for all accounts granted as part their work with us.
```